Sources/xcodeproj/Utils/ReferenceGenerator.swift

```diff
@@ -258,9 +258,14 @@ extension PBXObject {
     /// - Returns: object reference.
     func fixReference(identifiers: [String]) {
         if reference.temporary {
-            let identifiers = [String(describing: type(of: self))] + identifiers
-            let value = identifiers.joined(separator: "-").md5.uppercased()
-            reference.fix(value)
+            let typeName = String(describing: type(of: self))
+            let acronym = typeName
+                .replacingOccurrences(of: "PBX", with: "")
+                .replacingOccurrences(of: "XC", with: "")
+                .filter { String($0).lowercased() != String($0) }
+
+            let id = ([typeName] + identifiers).joined(separator: "-").md5.uppercased()
+            reference.fix("\(acronym)_\(id)")
```

Comment:
Why are we adding the acronym here? 
I'd add a comment with some examples of those references. It'll make it easier for someone reading the code in the future to understand the format.

Reply:
Cool, if people find it useful let's add it.